.azure/pipelines/helix-matrix.yml

```diff
@@ -0,0 +1,61 @@
+# We only want to run full helix matrix on master
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+
+variables:
+- ${{ if ne(variables['System.TeamProject'], 'internal') }}:
+  - name: _UseHelixOpenQueues
+    value: 'true'
+- ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+  - group: DotNet-HelixApi-Access
+  - name: _UseHelixOpenQueues
+    value: 'false'
```

Comment:
Do we need an internal pipeline run?

Reply:
I'm not sure, but if that's the case why do we run on internal today?